cli/pprof.go

```diff
@@ -11,47 +11,51 @@ import (
 	"github.com/filecoin-project/lotus/node/repo"
 )
 
-var PprofCmd = &cli.Command{
-	Name:   "pprof",
-	Hidden: true,
-	Subcommands: []*cli.Command{
-		PprofGoroutines,
-	},
+func PprofCmd() *cli.Command {
+	return &cli.Command{
+		Name:   "pprof",
+		Hidden: true,
```

Comment:
you need to obey this so it doesn't show up in docs, same as flags, check for `Hidden` and skip if `true`